src/DumpCommand.php

```diff
@@ -73,42 +73,29 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $scriptOptions = $input->getOption('script-options');
 
         // find all commands
-        $process = new Process($script . ' ' . $scriptOptions . ' list --raw | awk \'{if (NF>1) print $1 " " substr($0, index($0,$2)); else print $1}\'');
+        $process = new Process($script . ' ' . $scriptOptions . ' list --format=xml');
         $process->run();
         if (!$process->isSuccessful()) {
             throw new \RuntimeException($process->getErrorOutput());
         }
 
-        $rawCommands = explode("\n", $process->getOutput());
-        array_pop($rawCommands);
+        $xmlCommands = $process->getOutput();
+        $xml = simplexml_load_string($xmlCommands);
```

Comment:
simplexml MAY be disabled (http://www.php.net/manual/en/simplexml.installation.php using the `--disable-simplexml` option). Adding this extension as a requirement in `composer.json` would be the easiest way to deal with that. Considering this extension is enabled by default, I'd say that this is a safe solution. (http://www.php.net/manual/en/simplexml.installation.php#121170)

Reply:
sure, would be good to add that to the composer.json. Just a note though that this extension was already being used, but I'm happy to add it anyway

Reply:
Ha! So it was already missing. Hopefully this will be accepted and merged soon.